main.py

```diff
@@ -115,6 +115,14 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: string. The route to be redirected.
+        handler: callable or string. Custom handler method to be called.
+        defaults: dict or default. Default or extra keywords to be returned by this route.
+
+    Returns:
+        string. Returns a route that redirects /foo/ to /foo.
```

Comment:
Make it `str`

Reply:
After going through the code, I don't think this would return a string (not sure!). Maybe you can use `print, type, etc.`  and check or read the docs [here](https://webapp2.readthedocs.io/en/latest/api/webapp2_extras/routes.html) to find the type of the return value.

Reply:
It seems this function actually just redirects to a route and so does not return anything like a string but rather redirects to the route and executes the logic from the handler function.